scripts/exregional_run_post.sh

```diff
@@ -233,6 +235,49 @@ ${phy_file}
  KPO=47,PO=1000.,975.,950.,925.,900.,875.,850.,825.,800.,775.,750.,725.,700.,675.,650.,625.,600.,575.,550.,525.,500.,475.,450.,425.,400.,375.,350.,325.,300.,275.,250.,225.,200.,175.,150.,125.,100.,70.,50.,30.,20.,10.,7.,5.,3.,2.,1.,
  /
 EOF
+
+# 
+#-----------------------------------------------------------------------
+#
+# Let save the restart files if needed before run post.
```

Comment:
I suggest to add some comments to brief what the subsequent snippet does.

Reply:
Added a little more details on the action of this part of scripts.